contrib/IECoreUSD/src/IECoreUSD/PrimitiveAlgo.cpp

```diff
@@ -290,6 +291,48 @@ bool readPrimitiveVariables( const pxr::UsdSkelRoot &skelRoot, const pxr::UsdGeo
 		return false;
 	}
 
+	if( skinningQuery.HasBlendShapes() )
+	{
+		const pxr::UsdSkelAnimQuery &animQuery = skelQuery.GetAnimQuery();
+		if( !animQuery )
+		{
+			return false;
+		}
```

Comment:
Is it possible that these early outs mean we don't apply skinning when we could?

Reply:
For sure. What do you think is best? Apply skinning and don't apply blendshapes? With or without a warning?

Reply:
USD seems pretty good about spitting our warnings of its own, so maybe ours would be redundant...

Reply:
Yeah, I think it makes sense to leave the warnings to USD, and just apply as much as we can.

Reply:
Addressed in f874d8b344f916f6b605123403614d6d68f08e94